Google.Ads.GoogleAds/examples/Authentication/AuthenticateInAspNetCoreApplication/AuthenticateInAspNetCoreApplication.csproj

```diff
@@ -16,7 +16,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Condition="!Exists('..\..\..\src\Google.Ads.GoogleAds.csproj')" 
-        Include="Google.Ads.GoogleAds" Version="13.0.0" />
+        Include="Google.Ads.GoogleAds" Version="13.0.1" />
```

Comment:
Do you need to update `Authentication/AuthenticateInAspNetWebApplication/AuthenticateInAspNetWebApplication.csproj` as well?

Reply:
Done,